src/app/shared/stache/modules/image/image.component.html

```diff
@@ -0,0 +1 @@
+<img class="stache-image" [src]="imageSource" />
```

Comment:
Cool. Should this be a requirement for all of our components like this in the future?